lib/std/array_hash_map.zig

```diff
@@ -1924,7 +1924,7 @@ const IndexHeader = struct {
     }
 };
 
-test "basic hash map usage" {
+test AutoArrayHashMap {
```

Comment:
This test is pretty long. Not sure if we should make a shorter doctest just showing how to use the api.